NEWS.md

```diff
@@ -0,0 +1,3 @@
+# shiny.blueprint 0.1.0
+
+Initial release made available to the early access group.
```

Comment:
There was no "early access" group in this case :wink:

```suggestion
Initial (internal) release.
```

Reply:
Changed